packages/components/src/animate/stories/index.story.tsx

```diff
@@ -27,7 +27,10 @@ export const Default = Template.bind( {} );
 Default.args = {
 	children: ( { className } ) => (
 		<Notice className={ className } status="success">
-			<p>{ `No default animation. Use one of type = "appear", "slide-in", or "loading".` }</p>
+			<p>
+				No default animation. Use one of type = "appear", "slide-in", or
```

Comment:
Just using these quotes like this will create some `react/no-unescaped-entities` errors - can we use entities instead?

Reply:
I actually really dislike this rule. I have never found a good argument on why these entities could ever be a problem in a react JSX context, and I've never experienced any problems myself. I'd suggest just disabling them. Entity codes in the source are really annoying and hard to work with.

The official explanation is the following: "This rule prevents characters that you may have meant as JSX escape characters from being accidentally injected as a text node in JSX statements."

While this could theoretically happen and make someone confused for about 3 seconds, with code highlighting and linting this is virtually a non-issue IMHO, and the pain of working with something like `&#34;` in the source -rather than simply `"`- feels like a really bad trade-off to me (talking from experience).

Reply:
I see, and I sympathize with you about this one. However, linting rules often come with some trade-offs, it's tricky for them to serve 100% of the use cases well. I'm pretty open about where this goes, but it should be fixed somehow, even if you decide to do `eslint-disable` right there to avoid potential confusion and keep this more readable.

Reply:
This seems like a separate, unrelated change though. Mind doing it in another PR where we can discuss it separately?